react/src/components/ServingRouteErrorModal.tsx

```diff
@@ -71,12 +66,13 @@ const ServingRouteErrorModal: React.FC<Props> = ({
           {t("button.Close")}
         </Button>,
       ]}
-      {...modalProps}
+      {...props}
```

Comment:
I found a potential problem related to overriding props for modals. I'll make another PR to fix this.